packages/lit-starter-ts/src/test/my-element_test.ts

```diff
@@ -16,7 +17,9 @@ suite('my-element', () => {
   });
 
   test('renders with default values', async () => {
-    const el = await fixture(html`<my-element></my-element>`);
+    const el = await fixture(
+      html`<my-element></my-element>` as UncompiledTemplateResult
+    );
```

Comment:
I'm not clear why the changes in `my-element_test.ts` are needed? It seems outside the scope of `lit-html`?

Reply:
As part of this change I also fixed all the type errors that would arise if we changed TemplateResult to be MaybeCompiledTemplateResult